spec/controllers/devise_authy_controller_spec.rb

```diff
@@ -68,8 +68,8 @@
     describe "POST #verify_authy" do
       let(:verify_success) { 'approved' }
       let(:verify_failure) { 'invalid' }
-      let(:valid_verify_token) { rand(0..999999).to_s.rjust(6, '0') }
-      let(:invalid_verify_token) { rand(0..999999).to_s.rjust(6, '0') }
+      let(:valid_verify_token) { SecureRandom.rand(0..999999).to_s.rjust(6, '0') }
```

Comment:
Does this need to be placed at top of the file? Its already included in the Gemfile... 
```suggestion
      require 'securerandom' 
```

Reply:
Given that the tests are passing, I don't think adding it via require is necessary.